test/test.dart

```diff
@@ -375,22 +375,26 @@ void xtest(String? name, dynamic Function() testFunction, {dynamic skip, Map<Str
 
 Future<void>? baasSetup;
 
-Future<void> setupTests(List<String>? args) async {
+Future<void> setupTests(List<String>? args, {bool loggerTest = false}) async {
   arguments = parseTestArguments(args);
   testName = arguments["name"];
   setUpAll(() async => await (baasSetup ??= setupBaas()));
 
   setUp(() {
-    Realm.logger = Logger.detached('test run')
-      ..level = Level.ALL
-      ..onRecord.listen((record) {
-        testing.printOnFailure('${record.time} ${record.level.name}: ${record.message}');
-      });
-
+    final defaultLogger = Realm.logger;
```

Comment:
Could we isolate these changes only for `realm_logger_test.dart`. it seems to excessive we need to do these globally

Reply:
This change comes from another PR. They are done in order to print logs in the CI only when an error occurs.

Reply:
hm strange it shows as changes in this PR.